db/migrate/20221009105719_users.rb

```diff
@@ -0,0 +1,9 @@
+class Users < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :username
```

Comment:
I think we should also have added the real name. But otherwise, all your columns and their format seem correct. Great use of ActiveRecord build-in features!